sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py

```diff
@@ -38,6 +38,9 @@ class AzureAppConfigurationClient:
         :param str base_url: base url of the service
         :param credential: An object which can provide secrets for the app configuration service
         :type credential: azure.AppConfigConnectionStringCredential
+        :keyword Pipeline pipeline: If omitted, the standard pipeline is used.
+        :keyword HttpTransport transport: If omitted, the standard pipeline is used.
+        :keyword list[HTTPPolicy] policies: If omitted, the standard pipeline is used.
```

Comment:
`List[HTTPPolicy]`?

Reply:
This is to align with what's in azure.core.